Scripts/Game/ODIN/Editor/Containers/Attributes/ODIN_ToggleVisibilityEditorAttribute.c

```diff
@@ -0,0 +1,34 @@
+/**
+Entity Attribute for toggle unit visiblity
+*/
+[BaseContainerProps(), SCR_BaseEditorAttributeCustomTitle()]
+class ODIN_ToggleVisibilityEditorAttribute : SCR_ValidTypeBaseEditorAttribute
+{
+	override SCR_BaseEditorAttributeVar ReadVariable(Managed item, SCR_AttributesManagerEditorComponent manager)
+	{
+		
+		SCR_EditableEntityComponent editableEntity = SCR_EditableEntityComponent.Cast(item);
+		if (!editableEntity) return null;
```

Comment:
BI's convention seems to be to put the return in the next line instead.

Reply:
I can adopt their convention. I was unsure if we wanted "null checks" like that to be 1-line for brevity or 2-line for convention.

Reply:
I would say we stick to their convention

Reply:
Adopted the BI convention